system/goos/dependencies_linux.go

```diff
@@ -1,6 +1,70 @@
 package goos
 
+import (
+	"log"
+	"os/exec"
+
+	"github.com/ironstar-io/tokaido/system/console"
+)
+
 // CheckDependencies - Root executable
 func CheckDependencies() *string {
+	errDocker := CheckDocker()
+	errDockerCompose := CheckDockerCompose()
+	errUnison := CheckUnison()
+	errUnisonFsmonitor := CheckUnisonFsmonitor()
+
+	if (errDocker != nil) ||
+		(errDockerCompose != nil) ||
+		(errUnison != nil) ||
+		(errUnisonFsmonitor != nil) {
+		log.Fatalf("Unable to proceed. One or more dependency checks failed")
```

Comment:
Which dependency failed? Use might get lost with this message.

Also, are we not trying to auto-install missing dependencies for Linux? Too many variations with different packaging systems?